code/modules/spells/ability_types/realized.dm

```diff
@@ -61,7 +61,7 @@
 	cooldown_time = 30 SECONDS
 
 	var/debuff_range = 8
-	var/debuff_slowdown = 0.5 // Slowdown per use(funfact this was meant to be an 80% slow but I accidentally made it 20%)
+	var/debuff_slowdown = 0.2 // Slowdown per use(funfact this was meant to be an 80% slow but I accidentally made it 20%)
 /obj/effect/proc_holder/ability/lamp/Perform(target, mob/user)
```

Comment:
I don't know WHY but this ability no longer applies it's relevant debuff? Or maybe it never did. Please re-add that.